birdsongs/utils.py

```diff
@@ -66,9 +66,14 @@ def AudioPlay(obj):
 
 def Klicker(fig, ax):
     zoom_factory(ax)
-    ph = panhandler(fig, button=2)
+    pm = PanManager(fig, button=MouseButton.MIDDLE)
     klicker = clicker(ax, ["tini","tend"], markers=["o","x"], 
                       legend_bbox=(0.98, 0.98))# #legend_loc='upper right',
+
+    # hacky trick to keep the panmanager alive as long as the clicker is around
+    # without having to return another object
+    klicker._pm = pm
+
```

Comment:
I suspect that without this the panning behavior didn't used to work?

Reply:
Yes, I had some trouble getting it to work, and also with the title of the legend or trying to add a different label

Reply:
I tried to just use your suggestion, without a return in this function, but the clicker does not work. It looks like the klicker return must be done, in another case I cannot select the points. 
Thank you for your advice, they are very useful for me.